libdrgn/drgn.h

```diff
@@ -3429,6 +3429,15 @@ struct drgn_error *drgn_thread_object(struct drgn_thread *thread,
 struct drgn_error *drgn_thread_stack_trace(struct drgn_thread *thread,
 					   struct drgn_stack_trace **ret);
 
+/**
+ * Get name for the thread represented by @p thread.
+ *
+ * @param[out] ret Returned thread name. On success, it should be freed with
+ * free(). On error, it is NULL.
```

Comment:
Slight clarification:
```suggestion
 * @param[out] ret Returned thread name, or @c NULL if not found. On success, it
 * should be freed with free(). On error, it is not modified.
```